examples/qml_minimal/CMakeLists.txt

```diff
@@ -22,125 +22,208 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-if(NOT USE_QT5)
-    find_package(Qt6 COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner)
-endif()
-if(NOT Qt6_FOUND)
-    find_package(Qt5 5.15 COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner REQUIRED)
-endif()
-# ANCHOR_END: book_cmake_setup
-
-# ANCHOR: book_cmake_find_qmake
-# The path to the qmake executable path needs to be passed to the Rust
-# library's build script to ensure it uses the same installation of Qt as CMake.
-get_target_property(QMAKE Qt::qmake IMPORTED_LOCATION)
-# ANCHOR_END: book_cmake_find_qmake
-
-# ANCHOR: book_cmake_find_corrosion
-find_package(Corrosion QUIET)
-if(NOT Corrosion_FOUND)
-    include(FetchContent)
-    FetchContent_Declare(
-        Corrosion
-        GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
-        GIT_TAG v0.4.5
+if(BUILD_WASM)
```

Comment:
instead of putting this all behind an option, maybe instead try doing the following to make QmlImportScanner optional

```cmake
if(NOT USE_QT5)
    find_package(Qt6 COMPONENTS Core Gui Qml QuickControls2 OPTIONAL_COMPONENTS QmlImportScanner)
endif()
if(NOT Qt6_FOUND)
    find_package(Qt5 5.15 REQUIRED COMPONENTS Core Gui Qml QuickControls2 OPTIONAL_COMPONENTS QmlImportScanner)
endif()
```